apps/web/modules/settings/organizations/privacy/components/booking-report-entry-details-modal.tsx

```diff
@@ -0,0 +1,206 @@
+      showToast(t("booking_report_dismissed"), "success");
+      onClose();
+      reset();
+    },
+    onError: (error) => {
+      showToast(error.message, "error");
+    },
+  });
+
+  const onSubmit = (data: FormData) => {
+    if (!entry) return;
+
+    addToWatchlist.mutate({
+      reportIds: [entry.id],
+      type: data.blockType,
+    });
+  };
+
+  const handleDismiss = () => {
+    if (!entry) return;
+
+    dismissBookingReport.mutate({
+      reportId: entry.id,
+    });
+  };
+
+  const handleGoBack = () => {
+    onClose();
+    reset();
+  };
+
+  const reasonMap: Record<string, string> = {
+    SPAM: t("spam"),
+    DONT_KNOW_PERSON: t("dont_know_person"),
+    OTHER: t("other"),
+  };
+
+  if (!entry) return null;
+
+  return (
+    <Dialog open={isOpen} onOpenChange={onClose}>
+      <DialogContent enableOverflow>
+        <DialogHeader title={t("review_report") + ` - ${entry.bookerEmail}`} />
+        <form onSubmit={handleSubmit(onSubmit)}>
+          <div className="space-y-5">
+            <div className="bg-subtle rounded-xl p-1">
+              <h2 className="text-emphasis px-5 py-4 text-base font-semibold">{t("details")}</h2>
+              <div className="bg-default space-y-4 rounded-xl p-5">
+                <div>
+                  <label className="text-emphasis mb-1 block text-sm font-semibold">{t("email")}</label>
+                  <p className="text-subtle text-sm">{entry.bookerEmail}</p>
+                </div>
+
+                <div>
+                  <label className="text-emphasis mb-1 block text-sm font-semibold">{t("reason")}</label>
+                  <p className="text-subtle text-sm">{reasonMap[entry.reason] || entry.reason}</p>
+                </div>
+
+                <div>
+                  <label className="text-emphasis mb-1 block text-sm font-semibold">{t("reported_by")}</label>
+                  <p className="text-subtle text-sm">{entry.reporter?.email || "—"}</p>
+                </div>
+
+                <div>
+                  <label className="text-emphasis mb-1 block text-sm font-semibold">{t("description")}</label>
+                  <p className="text-subtle text-sm">{entry.description || t("no_description_provided")}</p>
+                </div>
+
+                <div>
+                  <label className="text-emphasis mb-1 block text-sm font-semibold">
+                    {t("related_booking")}
+                  </label>
+                  <Link href={`/booking/${entry.booking.uid}`}>
+                    <div className="text-subtle flex items-center gap-1 text-sm">
+                      {entry.booking.title}
+                      <Icon name="external-link" className="h-4 w-4" />
+                    </div>
+                  </Link>
+                </div>
+              </div>
+            </div>
+
+            <div>
+              <h2 className="text-emphasis text-base font-semibold">{t("block")}</h2>
```

Comment:
Can we make this look like in the designs? 

This PR: 
<img width="638" height="176" alt="Screenshot 2025-11-04 at 1 36 57 PM" src="https://github.com/user-attachments/assets/7b517242-0a5e-4cc6-b8c0-2ace1496ac79" />

Figma: 
<img width="548" height="154" alt="Screenshot 2025-11-04 at 1 37 08 PM" src="https://github.com/user-attachments/assets/92414e65-7644-4a1d-b699-b244b6074100" />